project-base/app/tests/FrontendApiBundle/Functional/Cart/ApplyPromoCodeToCartTest.php

```diff
@@ -333,4 +333,14 @@ private function applyPromoCodeToCart(string $promoCode, string $cartUuid = Cart
             'promoCode' => $promoCode,
         ]);
     }
+
+    /**
+     * @param \App\Model\Order\PromoCode\PromoCode $promoCode
+     * @param array $promoCodeData
+     */
+    private function assertPromoCode(AppPromoCode $promoCode, array $promoCodeData): void
+    {
+        self::assertEquals($promoCode->getCode(), $promoCodeData['code']);
+        self::assertEquals($promoCode->getDiscountType(), $promoCodeData['type']);
+    }
```

Comment:
This method is used in more than one test, dont you want to separate it into some helper or trait?

Reply:
ok, good idea - 18dfe1db88